doc/source/schedule-backup.rst

```diff
@@ -0,0 +1,51 @@
+.. _schedule: 
+
+Schedule backups
+==============================
+
+In |PBM| version 1.4.1 and earlier, we recommend using ``crond`` or similar services to schedule backup snapshots.
+
+.. important::
+
+   Before configuring ``crond``, make sure that you have :ref:`installed <install>` and :ref:`configured <initial-setup>` |PBM| to make backups in your database. Start a backup manually to verify this: :command:`pbm backup`. 
+
+The recommended approach is to create a :file:`crontab` file in the :dir:`/ect/cron.d` directory and specify the command in it. This simplifies server administration especially if multiple users have access to it.
+
+As an example, let's configure to run backup snapshots on 23:30 every Sunday. 
+
+The steps are the following:
+
+1. Create a ``crontab`` file. Let's name it ``pbm-backup``. 
+2. Specify the command in the file:
+   
+   .. code-block:: guess
+   
+      30 23 * * sun . /etc/default/pbm-agent; /usr/bin/pbm backup 
+
+ where the :file:`/etc/default/pbm-agent` is the environment variable file path for Debian and Ubuntu and the dot ``.`` equals to bash builtin ``source``.
+
+3. Verify that backups are running in :file:`/var/log/cron` or :file:`/var/log/syslog` logs:
+   
+   .. code-block:: guess
+   
+      $ grep CRON /var/log/syslog
+
+.. note:: 
+
+   Previous backups are not automatically removed from the backup storage. 
+   Therefore, timely remove them to clean up the storage. 
```

Comment:
```suggestion
   Previous backups are not automatically removed from the backup storage. You need to remove the oldest ones periodically to limit the amount of space used in the backup storage. The recommend solution is to run ``pbm delete-backup --older-than YYYY-MM-DD`` using a crontab entry similar to the one for ``pbm backup`` shown above.
```

Reply:
As discussed during standup, delete command should be like this 
`/usr/bin/pbm delete-backup -f --older-than $(date -d '-1 month' +\%Y-\%m-\%d)`